### PR TITLE
Bump Go to 1.21

### PR DESCRIPTION
**Pull request by liranr23**
Fixes: #751

### CODE REVIEW

**Comment by liranr23** [comment]
@ahadas based on the UUID change, it may happen that if a user had an OVA plan previously and will upgrade it to 2.6.1 containing this commit, he will need to change the IDs in the plan.

**Comment by bkhizgiy** [comment]
@liranr23 I think from the UI at least we should get a warning, since the ID will change the value in the inventory will also change and we should get `VM wasn't found` or something of a sort in the plan validations, otherwise yes its need to be manually changed.

**Comment by liranr23** [comment]
> @liranr23 I think from the UI at least we should get a warning, since the ID will change the value in the inventory will also change and we should get `VM wasn't found` or something of a sort in the plan validations, it still need to be change but we will have indication for that.

I think that will be tough to do.
1. it's possible that really something change and the plan isn't ready.
2. even if we drop the point above, we will need to to see which plan was ready and changed state while comparing to when did we upgrade.

i think the best solution will be documenting it. if an existing plan was in ready status and after the upgrade it changed, new plan creation(or editing) with the new ids required.

File: tests/suit/framework/ova.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func (r *OvaClient) LoadSourceDetails() (vm *OvaVM, err error) {
 		r.storageClass = "nfs-csi"
 	}
 
-	r.vmData.testVMId = "c5686650854d1e69b4123f4bf2e70fe1ed2a"
+	r.vmData.testVMId = "fdb4680df9160052f35e66c932e3fe5b62f5"
```
Comment (ahadas) [inline tests/suit/framework/ova.go:20]: need to look at our options and see if we cannot preserve those IDs

Comment (liranr23) [reply tests/suit/framework/ova.go:20]: that might be done if we are going to make them in another way. which means, it might change them anyway

Comment (ahadas) [reply tests/suit/framework/ova.go]: handled in #884

**Review by ahadas — APPROVED** [review]

**Pull request merged**